### DISCUSSION TITLE
Slack integration via Helm chart

### DISCUSSION THREAD

**Discussion by dyipon** [coroot/coroot#9] [Q&A]
Hi,

is there any way to enable/setup the Slack integration upon the Helm install process?

**Comment by apetruhin** [comment]
To set up integration with Slack, you need to create a Slack application and install it in your Slack workspace. It's all done on the Slack site ([docs](https://coroot.com/docs/coroot-community-edition/getting-started/alerting)).
Coroot uses the Slack application Bot Token to send you alerts.
There's not much we can do during the Helm install process.

Please, describe your usage of Coroot. Are you running multiple Coroot instances?

**Comment by dyipon** [comment]
hi! thanks for fast reply! slack already setuped, but i have to reinstall coroot again when I have to debug something, because after a few hours working it eats 3vcpu and 8gb ram. Is this normal?

**Comment by dyipon** [comment]
oh sorry, forgot to describe my coroot usage:
installed onto a 12 node cluster, via helm
my purpose is to detect/alert on SLO failures, and pod restart

**Comment by apetruhin** [comment]
Resources consumption depends on your infrastructure size (how many nodes, applications, instances, etc).
Which version of Coroot are you using? Please upgrade to the latest one (0.9.0).

BTW, you could join [Coroot Community Slack](https://join.slack.com/t/coroot-community/shared_invite/zt-1gsnfo0wj-I~Zvtx5CAAb8vr~r~vecyw) to see announcements and discuss features.

**Comment by dyipon** [comment]
updated, and joined to slack, thanks!

**Comment by dyipon** [comment]
Would be possible if coroot get the slack oath API key from a configmap or a secret?
